# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
# Define a message type for passing an array of points to the server and getting the merged points back

# Define the action goal message
string client_id
Point2D[] points